README.md

```diff
@@ -1 +1,49 @@
-# TO-DO-LIST
\ No newline at end of file
+## TO-DO-LIST
+Hello everyone, in this project I will be building a to-do-list app with webpack.
+# DESCRIPTION
+In this project, I will a list of items to be done will be generated dynamically from and array of object. Following the minimalist design template. Webpack will handle how the html, css and javascript file.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- VSCODE
+- Webpack
+
+## Live Demo
+
+[Live Demo Link]()
+
+# Prerequisite
+* Modern Browser.Example: Firefox, chrome.
+* Windows 7 and above, MacOs, linux and android.
+# SET UP
```

Comment:
- [ ] kindly add more steps here. Let's imagine that the person we are writing this is a non technical person. Highlight steps from cloning up to running the project on the user's local machine. `npm install` `git clone repo name`